components/sql/src/main/scala/pl/touk/nussknacker/sql/service/DatabaseEnricherInvokerWithCache.scala

```diff
@@ -33,19 +33,20 @@ class DatabaseEnricherInvokerWithCache(query: String,
   // TODO: cache size
   private val cache = Caffeine.newBuilder()
     .expireAfterWrite(cacheTTL)
-    .build[CacheKey, CacheEntry[queryExecutor.QueryResult]]
+    .buildAsync[CacheKey, CacheEntry[queryExecutor.QueryResult]]
```

Comment:
maybe lets extend some Tests just to be sure that caching works when is should

Reply:
it's actually testing a library.. but I've added test proving that it works

Reply:
+ it was already tested though - we do update on DB, and checking that enricher returns old value

Reply:
or maybe you mean testing other behaviour? if so, which one?